tests/unit/bots/test_bots.py

```diff
@@ -66,7 +66,10 @@ def test_select_owner_info(self):
             owner = self._generate_test_owner(with_bot=False)
             expected = AdapterAuthInformation(
                 token=Token(
-                    key="owner_token", refresh_token="refresh_token", secret=None
+                    key="owner_token",
```

Comment:
This is probably the most important file to check to ensure the entity_name logic makes sense